### ISSUE TITLE
Reduce the use of italics

### ISSUE THREAD

**Issue by brendanarnold** [DFE-Digital/polis-whitelabel#15]
Comment from the October Accessiblity Review ...

> You should avoid using italics where possible, this font formatting often makes
it harder for people with reading difficulties or minor visual impairments to
read.

We should look to remove the italics where the usage is purely stylistic. If emphasis is required consider using `<em>` tags with bold styling instead.

**Comment by patcon** [comment]
Agreed. As I understand, italics is harder to read for people with visual impairment, yes?

## Screenshot

<img width="834" alt="Screen Shot 2022-12-09 at 6 17 46 PM" src="https://user-images.githubusercontent.com/305339/206766920-36309912-3db0-4612-9af0-c82693480a70.png">

From https://pol.is/2demo
